test/generic.spec.js

```diff
@@ -0,0 +1,174 @@
+var fs = require('fs');
+var Promise = require('bluebird');
+var readFile = Promise.promisify(require('fs').readFile);
+var expect = require('chai').expect;
+var async = require('async');
+var Converter = require('csvtojson').Converter;
+var pg = require('pg');
+
+/*
+ This enforce driver to cast bigint (64-bits length) values into javascript (32-bit) integer.
+ We advice to do not use bigint data type. However this setting is useful for COUNT aggregations that
+ returns bigint.
+ Plese refer https://github.com/brianc/node-postgres/wiki/pg#pgdefaultsparseint8
+ */
+pg.defaults.parseInt8 = true;
+pg.defaults.poolSize = 2;
+
+var pgConnect = Promise.promisify(pg.connect, pg);
+
+// TODO set some generic config
+var config = {
+    db: {
+        connectionUrl: process.env.DATABASE_URL || 'postgres://realskill:realskill@localhost/sql-tasks'
```

Comment:
Even if someone would notice how database should be set we are supposed to have README explaining all necessary configuration dependencies because for me even tests are failing because I don't have correct relations(?). It is fine if you are planning to add in in later version but this version isn't finished without it.